TypeCobol/Compiler/CodeElements/Expressions/StorageArea.cs

```diff
@@ -70,6 +70,21 @@ public override string ToString()
         public virtual bool AcceptASTVisitor(IASTVisitor astVisitor) {
             return astVisitor.Visit(this) && this.ContinueVisitToChildren(astVisitor, SymbolReference,ReferenceModifier);
         }
+
+        public override bool Equals(object obj)
+        {
+            return Equals(obj as StorageArea);
+        }
+
+        public bool Equals(StorageArea storageArea)
+        {
+            return System.Object.ReferenceEquals(this, storageArea);
+        }
+
+        public override int GetHashCode()
+        {
+            return ToString().GetHashCode();
```

Comment:
Equals use ReferenceEquals, but GetHashCode use ToString().
Why ?

Reply:
The overrides of Equals/GetHashCode, hence IEquatable<StorageArea>, are not necessary; they have been removed.
https://github.com/TypeCobolTeam/TypeCobol/pull/1555/commits/77c907c8055e95bf877861ca4745d1033758792c